### ISSUE TITLE
Close menu on click?

### ISSUE THREAD

**Issue by TimWillaert** [kontentino/react-multilevel-dropdown#6]
I'm using this package to create a multi-level dropdown menu where the user can select their audio and video input devices for a video chat. However, when I click on one of the items in the Submenu, the menu doesn't close.
This is what my structure looks like:

![Screenshot_2](https://user-images.githubusercontent.com/43609662/119271114-6080ae80-bc00-11eb-956d-32ce9cbae746.jpg)

Is this the default behavior?

**Comment by suraj-pgi** [comment]
You can achieve with some workaround.
Add id to Dropdown tag and by using that id you can trigger click `document.getElementById("sameId").click();`
and menu will get closed

**Comment by filippofilip95** [comment]
Hi We have released new version 3.0.0 where you can handle it by reference on dropdown like this 
 
 ```js
 export const CloseOnItemClick = () => {
    const ref = useRef(null);


    function onClick(e) {
        if (ref.current) {
            ref.current.toggle(e);
        }
    }

    return (
        <UIDropdown title='Dropdown example' ref={ref}>
            <UIDropdown.Item onClick={onClick}>
                Close on Click
            </UIDropdown.Item>
            ... more code
 ```


Now it's also available in Storybook [here](https://kontentino.github.io/react-multilevel-dropdown/?path=/story/dropdown-usages--close-on-item-click)

or directly at source code of story [here](https://github.com/kontentino/react-multilevel-dropdown/blob/master/src/DropdownUsage.stories.js#L89 )

**Comment by vinayakvvg** [comment]
Hey @filippofilip95
 
The storybook(demo + source code) links mentioned have expired, can you check and refresh them
